pkg/resource/interfaces.go

```diff
@@ -233,6 +243,8 @@ type CompositeClaim interface {
 	CompositionRevisionReferencer
 	CompositeResourceReferencer
 	LocalConnectionSecretWriterTo
+	// TODO(turkenh): LocalConnectionDetailsPublisherTo?
```

Comment:
I don't think that would be needed under the new design. I'm pretty sure the difference between `Connection` and `LocalConnection` was that one had a namespaced ref (e.g. with name and namespace) while the other was just a name (implying writing to the claim's namespace).

Reply:
Yeah, I can confirm this with my testing so far.